docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,306 @@
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
    "source": [
     "Reusing an already built reverse proxy image is done when:\n",
     "- Redeploying a function that built the reverse proxy once and has `application.status.container_image` enriched.\n",
     "- It was already built manually with `mlrun.runtimes.ApplicationRuntime.deploy_reverse_proxy_image()`.\n",
     "\n",
-    "Using one of the above options can help minimize redundant builds and streamline the development cycle. "
-   ],
+    "Using one of the above options can help minimize redundant builds and streamline the development cycle."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "### combining application with serving function\n",
+    "In this example, we deploy a serving function and use it with our application.\n",
+    "</br></br>\n",
+    "<u>Serving creation</u>:</br>\n",
+    "First, we will create our model file and save it as a .py file."
```

Comment:
"### Combining the application with a serving function\n",
"This example demonstrates deploying a serving function and using it with the application.\n",
    "</br></br>\n",
    "<u>Serving creation</u>:</br>\n",
    "First, create the model file and save it as a .py file."